### ISSUE TITLE
ERROR: Level::AllocEdict: no free edicts

### ISSUE THREAD

**Issue by bulldozerecske** [openmoh/openmohaa#143]
OpenMohaa crashed on AA - m5l3 when Flak 88 was shooting at me.

```
Can't read sound file sound/null.mp3
WARNING: Failed to load sound sound/null.wav!
WARNING: could not find sound/null.wav - using default
Warning: Couldn't find view model animation unarmed_putaway
z:    params 97.388448.140
z:    params 95.953440.248
z:    params 95.109435.608
z:    params 93.154424.853
********************
ERROR: Level::AllocEdict: no free edicts
********************
----- Server Shutdown (Server crashed: Level::AllocEdict: no free edicts) -----
==== ShutdownGame ====
------ Unloading Game ------
---------------------------
Channel memory manager started
------ Unloading ClientGame ------
Channel memory manager started
Channel memory manager started
Channel memory manager started
------ Unloading ClientGame ------
Channel memory manager started
Channel memory manager started
------ Unloading ClientGame ------
Channel memory manager started
Channel memory manager started
Setting up Shaders
WARNING: R_FindImageFile could not find 'textures/sprites/white.tga' in shader 'white'
```

**Comment by bulldozerecske** [comment]
OpenMohaa dedicated server crashed on Omaha Beach (without players):

```
SV packet xxx.xxx.xxx.xxx : getstatus
SV packet xxx.xxx.xxx.xxx : getstatus
SV packet xxx.xxx.xxx.xxx : getstatus
********************
ERROR: Level::AllocEdict: no free edicts
********************
----- Server Shutdown (Server crashed: Level::AllocEdict: no free edicts) -----
==== ShutdownGame ====
------ Unloading Game ------
---------------------------
```

**Comment by smallmodel** [comment]
Fixed in commit 07ec6adecb7ddb3f4b4ee5ce2a98b2804ec9cef8
